packages/suite/src/components/suite/AppNavigation.tsx

```diff
@@ -220,16 +220,17 @@ const isSecondaryMenuOverflown = ({ primary, secondary, wrapper }: MenuWidths) =
     primary + secondary >= wrapper;
 
 export const AppNavigation = ({ items, primaryContent, maxWidth, inView }: Props) => {
-    const theme = useTheme();
     const [condensedSecondaryMenuVisible, setCondensedSecondaryMenuVisible] =
         useState<boolean>(false);
     const wrapper = useRef<HTMLDivElement>(null);
     const primary = useRef<HTMLDivElement>(null);
     const secondary = useRef<HTMLDivElement>(null);
-    const { routeName, screenWidth } = useSelector(state => ({
-        routeName: state.router.route?.name,
-        screenWidth: state.resize.screenWidth,
-    }));
+
+    const routeName = useSelector(state => state.router.route?.name);
+    const { screenWidth } = useSelector(state => state.resize);
+    const { selectedAccount } = useSelector(state => state.wallet);
```

Comment:
nit: to prevent unnecessary re-renders
```suggestion
    const screenWidth = useSelector(state => state.resize.screenWidth);
    const selectedAccount = useSelector(state => state.wallet.selectedAccount);
```

Reply:
Right, fixed.